docs/rfcs/README.md

```diff
@@ -0,0 +1,31 @@
+# RFC Process
+
+The RFC process for the OpenTelemetry Operator exists so that community members can effectively review and understand
+major decisions made by the Operator SIG. The RFC process also allows users to comment asynchronously on design
+decisions.
+
+We aim to improve the experience for using OpenTelemetry tools in a Kubernetes
+environment. Proposals here should only focus on improving that experience by composing existing
+OpenTelemetry components such as the OpenTelemetry Collector, OpenTelemetry Instrumentation, OpAMP, etc.
+If a proposal requires new components to exist that do not logically fit within that mission, it's recommended to
+open proposals with the proper SIGs.
+
+## Process
+
+1. Copy and fill the template.md document
+2. Open a PR to get initial feedback on RFC
+3. The RFC must be brought before the SIG at least once prior to merge
```

Comment:
Do you mean that the RFPC must be discussed at the SIG meeting before merge? Maybe rephrasing the sentence would help to understand it?